src/pages/index.js

```diff
@@ -262,22 +263,95 @@ const IndexPage = (props) => (
+              />
+              <CardText>
+                <p className={testimonialStyles.cardText}>
+                "Careers in Code is a coding bootcamp that teaches computer programming to women and minorities to help..."
+                </p>
+              </CardText>
+              <CardTitle>
+                <p className={testimonialStyles.cardTitle}>
+                Introducing the Second Cohort of Careers in Code Students!
+                </p>
+              </CardTitle>
+            </Card>
+          </Col>
+          </a>
+
+          <a href="https://hackupstate.medium.com/announcing-our-career-coach-laura-thorne-f628f063ef02" target="_b">
+          <Col sm="4">
+            <Card body className={("shadow-sm", testimonialStyles.cardStyle)}>
+              <CardImg
+                top
+                width="100%"
+                src={"https://miro.medium.com/max/1400/0*x5V_ueee_400JYrx"}
+                alt=""
+                className={testimonialStyles.cardImg}
+              />
+              <CardText>
+                <p className={testimonialStyles.cardText}>
+                  "We’re very excited to announce that Laura Thorne has joined our team as our Career Coach for our second..."
+                </p>
+              </CardText>
+              <CardTitle>
+                <p className={testimonialStyles.cardTitle}>
+                Announcing Our Career Coach, Laura Thorne!
+                </p>
+              </CardTitle>
+            </Card>
+          </Col>
+          </a>
+
+          <a href="https://hackupstate.medium.com/technology-is-changing-the-roles-of-teachers-and-students-c34c54f24531" target="_b">
+          <Col sm="4">
+            <Card body className={("shadow-sm", testimonialStyles.cardStyle)}>
+              <CardImg
+                top
+                width="100%"
+                src="https://miro.medium.com/max/2560/0*z_Mr56BzDN726WBN"
+                alt="https://miro.medium.com/max/2560/0*z_Mr56BzDN726WBN"
+                className={testimonialStyles.cardImg}
+              />
+              <CardText>
+                <p className={testimonialStyles.cardText}>
+                  "Technology is changing the duties of students and teachers in the classroom. Students are doing more..."
+                </p>
+              </CardText>
+              <CardTitle>
+                <p className={testimonialStyles.cardTitle}>
+                  Technology Is Changing The Roles Of Teachers And Students
+                </p>
+              </CardTitle>
+            </Card>
+          </Col>
+          </a>
+        </CardDeck>
+      </Row>
+
+      <Row className={testimonialStyles.buttonRow}>
+        <Col>
+        <a
+              href={"https://hackupstate.medium.com/"}
+              rel="noopener noreferrer"
+              className={testimonialStyles.btnLink}
+            >
+          <Button size="md" className={testimonialStyles.blueButton}>
```

Comment:
This should open in a new tab. See examples above for who to to do. You should be able to put `target="_blank"`

Reply:
This is resolved now due to adding target="_blank". Nice work!